po/pt-BR.po

```diff
@@ -1068,60 +1087,74 @@ msgstr "## Suposições"
 #: src/welcome.md:39
 #, fuzzy
 msgid ""
-"The course assumes that you already know how to program. Rust is a statically\n"
-"typed language and we will sometimes make comparisons with C and C++ to better\n"
+"The course assumes that you already know how to program. Rust is a "
```

Comment:
I think this change is caused by a change in the tool used to write the `.po` file? Would you be able to normalize this so that the diff better represents the changes made?

Reply:
Basically, the docs.rs/polib library we use when generating the `messages.pot` file is not very clever about how it breaks likes. Your Poedit did a much better job in the version that is committed.

Reply:
Good point, resubmitting with Poedit formatting!

Reply:
Thanks!